src/business_logic/transaction/transactions.rs

```diff
@@ -0,0 +1,45 @@
+use crate::{
+    business_logic::{
+        execution::objects::TransactionExecutionInfo,
+        state::state_api::{State, StateReader},
+    },
+    definitions::general_config::StarknetGeneralConfig,
+    utils::Address,
+};
+
+use super::{
+    error::TransactionError,
+    objects::{internal_deploy::InternalDeploy, internal_invoke_function::InternalInvokeFunction},
+};
+
+pub(crate) enum Transaction {
```

Comment:
Please add the rest of the transactions

Reply:
i think that It is not necessary yet, as they are not used and i am not sure that all of them shared this methods.